src/ecredis.erl

```diff
@@ -459,6 +465,18 @@ handle_ask(#query{command = Command, retries = Retries,
     end.
 
 -spec handle_error(Error :: atom(), Query :: #query{}) -> {[#query{}], [#query{}]}.
+handle_error(no_connection,
+        #query{query_type = qmn, cluster_name = ClusterName,
+            command = [Command | _], retries = Retries} = Query) ->
+    {ok, RemapNewVersion} = remap_cluster(Query),
+    case get_destination(ClusterName, Command) of
+        {ok, _Slot, Pid, Version} ->
+            {[], [Query#query{pid = Pid, version = Version, retries = Retries + 1}]};
```

Comment:
Will this execute all the qmn queries again?

Reply:
Basically a qmn query is split into N queries. Each query contains M redis commands that should execute on the same node. When we don't have a connection to this node we get just one {error, no_connection} and so here we remap and try to find a Pid again.